cocos/audio/android/PcmAudioService.h

```diff
@@ -71,6 +71,7 @@ class PcmAudioService
     int _numChannels;
     int _sampleRate;
     int _bufferSizeInBytes;
+    bool _isInitialised;
```

Comment:
No need to add a new member variable for this purpose. It wastes 4 bytes because struct alignment.
I think checking `_playItf` could still resolve this issue.
```c++
void PcmAudioService::pause()
{
    if (_playItf != nullptr && (*_playItf) != nullptr)
    {
        SLresult r = (*_playItf)->SetPlayState(_playItf, SL_PLAYSTATE_PAUSED);
        SL_RETURN_IF_FAILED(r, "PcmAudioService::pause failed");
    }
}

void PcmAudioService::resume()
{
    if (_playItf != nullptr && (*_playItf) != nullptr)
    {
        SLresult r = (*_playItf)->SetPlayState(_playItf, SL_PLAYSTATE_PLAYING);
        SL_RETURN_IF_FAILED(r, "PcmAudioService::resume failed");
    }
}
```